gas-station/typescript/src/sponsored_examples.ts

```diff
@@ -1,163 +1,240 @@
+//   sui: nodeClient,
+//   build: async (txb) => {
+//     txb.transferObjects(
+//       [txb.object(OBJECT_TO_SEND)],
+//       txb.pure(RECIPIENT_ADDRESS)
+//     );
+//   }
+// });
+
+
+
+//  Merge one or more smaller coins into another, destroying the small coin(s)
+//    and increasing the value of the large one.
+//
+//
+// const TARGET_COIN = "{{targetCoin}}";
+// const COIN_SOURCE1 = "{{coinSource1}}";
+// const COIN_SOURCE2 = "{{coinSource2}}";
+// const gaslessPayloadBase64 = await buildGaslessTransactionBytes({
+//   sui: nodeClient,
+//   build: async (txb) => {
+//     txb.mergeCoins(txb.object(TARGET_COIN), [
+//       txb.object(COIN_SOURCE1),
+//       txb.object(COIN_SOURCE2),
+//     ]);
+//   }
+// });
+
+
+// -- Invisible Wallet: sign, sponsor, and execute as separate steps -- //
+
+// 3. Use a pre-existing Shinami Invisible Wallet as the sender
+// import { KeyClient, WalletClient } from "@shinami/clients";
+
+// const WALLET_SERVICES_KEY = "{{walletAccessKey}}";
+// const INVISIBLE_WAL_SECRET = "{{walletSecret}}";
+// const INVISIBLE_WAL_ID = "{{walletId}}}";
+
+// const walletClient = new WalletClient(WALLET_SERVICES_KEY);
+// const SENDER_ADDRESS = await walletClient.getWallet(INVISIBLE_WAL_ID);
+
+// 8. Sign the full transaction payload with the sender's key.
+// const keyClient = new KeyClient(WALLET_SERVICES_KEY);
+
+// const sessionToken = await keyClient.createSession(INVISIBLE_WAL_SECRET);
+// const senderSig = await walletClient.signTransactionBlock(
+//   INVISIBLE_WAL_ID,
+//   sessionToken,
+//   sponsoredResponse.txBytes
+// );
+
+// const WALLET_SERVICES_KEY = "{{walletAccessKey}}";
+// const INVISIBLE_WAL_SECRET = "{{walletSecret}}";
+// const INVISIBLE_WAL_ID = "{{walletId}}}";
+
+// -- Invisible Wallet: sponsor, sign, and execute in one method call -- //
+// import { 
+//   KeyClient, 
+//   WalletClient, 
+//   createSuiClient, 
+//   buildGaslessTransactionBytes 
+// } from "@shinami/clients";
+
+// // 1. Copy your Node and Wallet Services Testnet Api Key Values
+// const NODE_TESTNET_ACCESS_KEY = "sui_testnet_bb0ae4d25b7e51a98c8fcb267babc1b6";
+// const WALLET_SERVICES_ONLY_KEY = "3ccf23b29a99cb4d4c2974e590575f23";
+// const ALL_SERVICES_TESTNET_KEY = "sui_testnet_a683fdb22331e5a9ef146b4dc2f5cf67";
+
+// // 2. Set up your Node client
+// const nodeClient = createSuiClient(NODE_TESTNET_ACCESS_KEY);
+
+// // 3. Generate the TransactionKind for sponsorship as a Base64 encoded string
+// const gaslessPayloadBase64 = await buildGaslessTransactionBytes({
+//   sui: nodeClient,
+//   build: async (txb) => {
+//     txb.moveCall({
+//       target: "0xfa0e78030bd16672174c2d6cc4cd5d1d1423d03c28a74909b2a148eda8bcca16::clock::access",
+//       arguments: [txb.object('0x6')]
+//     });
+//   }
+// });
+
+// // 4. Set the Id and Secret values for a previously created Invisible Wallet
+// const INVISIBLE_WAL_SECRET = "akeuy32wekd";
+// const INVISIBLE_WAL_ID = "aaron";
```

Comment:
replace these with the tutorial stubs e.g. `{{walletSecret}}`

Reply:
I ended up moving these to the Invisible Wallet tutorial so I didn't have the same code twice, and linking to that. I'll make that update in those examples